Neos.Neos/Classes/Domain/Model/FusionRenderingStuff.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Domain\Model;
+
+use Neos\Fusion\Core\FusionGlobals;
+
+final readonly class FusionRenderingStuff
```

Comment:
we dislike this naming

Reply:
@kitsunet do you think a model like this makes sense?

I think yes, as those are all the ingredients/recipe for a sucessfull neos node rendering